masterlist.yaml

```diff
@@ -226,6 +244,8 @@ common:
         text: 'Você aparenta estar a utilizar **%1%**, mas não ativou o patch de compatibilidade para este mod. Um patch de compatibilidade está disponível na página mod deste plugin.'
       - lang: pt_BR
         text: 'Você aparenta estar usando **%1%**, mas não habilitou o patch de compatibilidade para este mod. Um patch de compatibilidade é provido na página desse plugin.'
+       - lang: pl
```

Comment:
The indentation is off here and needs to be corrected. The rest of this pull is good.